packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -354,40 +357,64 @@ describe('Force Project Create', () => {
       ]);
       assert.file([
         path.join(
-          getRootWorkspacePath(),
+          workspaceUtils.getRootWorkspacePath(),
           'TestProject',
           'scripts',
           'apex',
           'hello.apex'
         )
       ]);
       assert.file([
-        path.join(getRootWorkspacePath(), 'TestProject', 'README.md')
+        path.join(
+          workspaceUtils.getRootWorkspacePath(),
```

Comment:
workspaceUtils.getRootWorkspacePath() is called many times here.  How about adding a new const and set it to workspaceUtils.getRootWorkspacePath().

Reply:
good call. done